docs/tutorials/ionq/binary_paintshop.ipynb

```diff
@@ -0,0 +1,421 @@
+{
```

Comment:
<p><code>remote_host</code> is no  longer needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>